cloudinary_cli/modules/sync.py

```diff
@@ -160,6 +146,36 @@ def push(self):
         if upload_errors:
             raise Exception("Sync did not finish successfully")
 
+    def pull(self):
+        """
+        Pulls changes from the Cloudinary folder to the local folder.
+        """
+        download_results = {}
+        download_errors = {}
+        if not self._handle_unique_local_files():
+            return False
+
+        files_to_pull = self.unique_remote_file_names | self.out_of_sync_remote_file_names
```

Comment:
log from which source we're pulling

Reply:
@strausr , it is logged at the very beginning of the execution of the `sync` command